### ISSUE TITLE
No longer possible to reach the "new notebook" button via shift-tab when scrolled down in the notebook list

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#14801]
### Operating system

Linux

### Joplin version

3.6.6

### Desktop version info

Joplin 3.6.6 (prod, linux)

Device: linux, 12th Gen Intel(R) Core(TM) i7-1260P
Client ID: 6ef78bbb20114ae48a4c58bba7393d22
Sync Version: 3
Profile Version: 49
Keychain Supported: No
Alternative instance ID: -
Sync target: (None)
Editor: Markdown

Revision: 3223d9c

Backup: 1.5.1
Freehand Drawing: 4.3.0

### Current behaviour

1. Focus a notebook near the end of the notebook list.
2. Press <kbd>shift</kbd>-<kbd>tab</kbd>.
3. Observe that focus **does not** move to the "new notebook" button.
4. Continue pressing <kbd>shift</kbd>-<kbd>tab</kbd>.
5. Observe that focus wraps around, but never reaches the (offscreen) "new notebook" button.

### Expected behaviour

- Scrolling down in the notebook list should not affect the page's focus order.
- It should be possible to reach the "new notebook" button by pressing <kbd>shift</kbd>-<kbd>tab</kbd>, even when scrolled down in the notebook list, as was the case in Joplin v3.6.4.

**Comment by personalizedrefrigerator** [comment]
# Old behavior (v3.6.4):

Pressing <kbd>shift</kbd>-<kbd>tab</kbd> from the notebook list focused the "new notebook" button and scrolled it into view:

https://github.com/user-attachments/assets/3754b923-7c89-4922-8508-539ae0b51fe0

# New behavior (v3.6.6)

Pressing <kbd>shift</kbd>-<kbd>tab</kbd> from the notebook list only focuses the "new notebook" button if "new notebook" is onscreen:

https://github.com/user-attachments/assets/da20baf3-6326-4d6e-8aea-2429252c5fad

**Comment by personalizedrefrigerator** [comment]
Possibly related to: https://github.com/laurent22/joplin/pull/14667

**Comment by laurent22** [comment]
If there are two issues related to this pull request I suggest we revert. It's really not important enough to break what was working fine

**Comment by laurent22** [comment]
It has now been reverted in ea4efa6a16b9c6208d7ad5d911216089c72be8c4

**Comment by personalizedrefrigerator** [comment]
> It has now been reverted in https://github.com/laurent22/joplin/commit/ea4efa6a16b9c6208d7ad5d911216089c72be8c4

Closing as resolved by the revert.
